manage.py

```diff
@@ -198,6 +247,66 @@ def set_database_configuration(
         )
 
 
+@manager.command
+@manager.option("-d", "--default_database", help="Default data source")

```

Comment:
``--default_database`` This is misleading since the datasource is not necessary a database.

Reply:
I will change the variable to be data_source